overlord/hookstate/ctlcmd/ctlcmd_test.go

```diff
@@ -112,3 +112,15 @@ func (s *ctlcmdSuite) TestHiddenCommand(c *C) {
 	// mock-hidden is not in the help message
 	c.Check(err.Error(), Not(testutil.Contains), "  mock-hidden\n")
 }
+
+func (s *ctlcmdSuite) TestRootRequiredCommandFailure(c *C) {
+	_, _, err := ctlcmd.Run(s.mockContext, []string{"start"}, 1000)
+
+	c.Check(err, FitsTypeOf, &ctlcmd.ForbiddenCommandError{})
+	c.Check(err.Error(), Equals, `cannot use "start" with uid 1000, try with sudo`)
+}
+
+func (s *ctlcmdSuite) TestRunNoArgsFailure(c *C) {
+	_, _, err := ctlcmd.Run(s.mockContext, []string{}, 0)
+	c.Check(err, NotNil)
+}
```

Comment:
It would be nice to have a test for the `-h` flag

Reply:
Indeed.

Reply:
We do for some commands (like`set`) https://github.com/snapcore/snapd/blob/1c24c84307a5732ec06eeea769e48aa6f7a2ba8b/overlord/hookstate/ctlcmd/set_test.go#L110

Reply:
Thanks for adding the tests!